addons/mail/static/src/discuss/message_pin/common/pinned_messages_panel.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
     <t t-name="discuss.PinnedMessagesPanel">
         <ActionPanel title="title">
-            <MessageCardList emptyText="emptyText" messages="messagePinService.getPinnedMessages(props.thread)" thread="props.thread" mode="'pin'" onClickUnpin="(message) => this.onClickUnpin(message)"/>
+            <MessageCardList emptyText="emptyText" messages="props.thread.pinnedMessages" thread="props.thread" mode="'pin'" onClickUnpin="(message) => message.unpin()"/>
```

Comment:
`onClickUnpin` props can be removed and the component just immediately handle click as `message.unpin()`